src/types/ChangeInfo.ts

```diff
@@ -20,6 +20,11 @@ export interface ChangeInfo extends ChangeFileInfo {
   commit: string;
 }
 
+export type ChangeInfoMultiple = {
```

Comment:
nit, can you use an interface here to match the convention in this repo?

Reply:
Good call, not sure what I was thinking. Ha.